cohorts/survival.py

```diff
@@ -76,6 +80,38 @@ def plot_kmf(df,
         print_as_title: bool, optional, whether or not to print text
           within the plot's title vs. stdout, default False
     """
+    # if strata not None, call plot_kmf for each of the strata
+    if strata_col:
+        # want this at the top of the function!
+        arglist = locals() ## save input args for reuse
+        # create axis / subplots
+        if ax is not None:
+            raise ValueError('ax not supported with stratified analysis.')
+        n_strata = len(df[strata_col].unique())
+        f, ax = plt.subplots(n_strata, sharex=True)
+        a = 0 ## index for ax
+        # create results dict to hold per-strata results
+        results = dict()
+        # update strata_col to be None for each sub-call
+        arglist['strata_col'] = None
+        # call plot_kmf for each strata, using same args
+        for strat_name, strat_df in df.groupby(strata_col):
```

Comment:
You could do `for a, (strat_name, strat_df) in enumerate(df.groupby(strata_col)):` to avoid declaring and incrementing `a`, but up to you if that's less clear.

Maybe call it `i`?